pages/documentation/pythnet-price-feeds/price-pusher.mdx

```diff
@@ -0,0 +1,10 @@
+# Price Pusher
```

Comment:
Moved out of EVM docs.

note: this will need to be renamed to scheduler, but let's do that in a separate PR because it's going to touch a lot of things.